scene/gui/popup_menu.cpp

```diff
@@ -88,8 +88,16 @@ String PopupMenu::bind_global_menu() {
 				for (int j = 0; j < events.size(); j++) {
 					Ref<InputEventKey> ie = events[j];
 					if (ie.is_valid()) {
-						ds->global_menu_set_item_accelerator(global_menu_name, index, ie->get_keycode_with_modifiers());
-						break;
+						if (ie->get_physical_keycode() == Key::NONE && ie->get_keycode() == Key::NONE && ie->get_key_label() != Key::NONE) {
```

Comment:
I wonder if this logic should be a method in InputEventKey, seems like it might often be needed to figure out what's the "active" combination.

This would also help documenting it so all the magic around handling of NONE values can be made explicit.

WDYT?

Reply:
Usually it's handled by action or event matching, and it's not necessary to do it manually, but in this specific case it's converted to raw key value since handling is done by OS instead of Godot event processing.

Reply:
I'd at least suggest a helper method in this class as it's repeated so many times and looks identical in each case

Reply:
Added a helper method.